ReactAndroid/src/main/java/com/facebook/react/bridge/JSDebuggerWebSocketClient.java

```diff
@@ -118,7 +118,6 @@ public void executeApplicationScript(
       for (String key : injectedObjects.keySet()) {
         jg.writeObjectField(key, injectedObjects.get(key));
       }
-      jg.writeEndObject();
```

Comment:
the line right after is 
```java
sendMessage(requestID, endMessageObject(jg));
```
where 'endMessageObject' is defined as 
```java
private String endMessageObject(JsonGenerator jg) throws IOException {
    jg.writeEndObject(); // <---- already called :)
    jg.flush();
    return ((StringWriter) jg.getOutputTarget()).getBuffer().toString();
  }
```
just like 
```java
public void prepareJSRuntime(JSDebuggerCallback callback) {
    int requestID = mRequestID.getAndIncrement();
    mCallbacks.put(requestID, callback);

    try {
      JsonGenerator jg = startMessageObject(requestID);
      jg.writeStringField("method", "prepareJSRuntime");
// <---- doesn't need 'jg.writeEndObject();' here 
      sendMessage(requestID, endMessageObject(jg));
    } catch (IOException e) {
      triggerRequestFailure(requestID, e);
    }
  }
```